pkg/motor/create_account.go

```diff
@@ -75,6 +72,7 @@ func (mtr *motorNodeImpl) CreateAccount(request mt.CreateAccountRequest) (mt.Cre
 	}
 
 	// create vault
+	mtr.callback.OnMotorEvent("Setting up Account Vault", false)
```

Comment:
References to the value represented by each string should be made through some kind of strongly typed value / constant